package/MDAnalysis/coordinates/XDR.py

```diff
@@ -197,7 +197,7 @@ def _load_offsets(self):
         try:
             with fasteners.InterProcessLock(lock_name) as filelock:
                 pass
-        except PermissionError:
+        except (PermissionError, OSError):
```

Comment:
According to the [Python docs](https://docs.python.org/3/library/exceptions.html#os-exceptions), `PermissionError` is a subclass of `OSError`, so your code is currently equivalent to only checking for `OSError`.
I'd advise against a simple check for `OSError` only, though, as this exception class covers many kinds of OS-related errors.
To narrow down the cause of the exception, it might be better to catch `OSError`, then check if its a `PermissionError` **or** the `errno` member of the exception is the one you encountered. If so, raise the warning as before, otherwise just `raise` for the exception to bubble up.
For checking the errno, there should be a suitable constant in the [`errno` module of the standard library](https://docs.python.org/3/library/errno.html) so you don't have to hard-code numbers here.